@xen-orchestra/backups/formatVmBackups.mjs

```diff
@@ -25,8 +25,8 @@ function formatVmBackup(backup) {
       name_description: backup.vm.name_description,
       name_label: backup.vm.name_label,
     },
-    differencingVhds: Object.values(backup.isVhdDifferencing).filter(t => t).length,
-    dynamicVhds: Object.values(backup.isVhdDifferencing).filter(t => !t).length,
+    differencingVhds: Object.values(backup.isVhdDifferencing ?? {}).filter(t => t).length,
+    dynamicVhds: Object.values(backup.isVhdDifferencing ?? {}).filter(t => !t).length,
```

Comment:
This approach is lying to the client by saying there are 0 differencing and dynamic VHDs, could it cause issues in the future?

We could also return `undefined` when this info is not known.